README.md

```diff
@@ -159,6 +160,10 @@
 * [rust-nix-templater](https://github.com/yusdacra/rust-nix-templater) - Generates Nix build and development files for Rust projects.
 * [rust-overlay](https://github.com/oxalica/rust-overlay) - Pure and reproducible nix overlay of binary distributed Rust toolchains.
 
+### Scala
+
+* [sbt-derivation](https://github.com/zaninime/sbt-derivation) -  mkDerivation for sbt, similar to buildGoModule
```

Comment:
To fit the "awesome list" style guide, I'd suggest editing this line slightly, adding an "A" and a period, and some pre-formatted blocks:

```text
A `mkDerivation` for sbt, similar to `buildGoModule`.
```

Reply:
I copied verbatim from the repo's description, but I'll keep this in mind :)